docs/modules/ROOT/pages/access-control.adoc

```diff
@@ -202,3 +134,132 @@ TIP: A recommended configuration is to grant both roles to a secure governance c
 Operations executed by the xref:api:governance.adoc#TimelockController[`TimelockController`] are not subject to a fixed delay but rather a minimum delay. Some major updates might call for a longer delay. For example, if a delay of just a few days might be sufficient for users to audit a minting operation, it makes sense to use a delay of a few weeks, or even a few months, when scheduling a smart contract upgrade.
 
 The minimum delay (accessible through the xref:api:governance.adoc#TimelockController-getMinDelay--[`getMinDelay`] method) can be updated by calling the xref:api:governance.adoc#TimelockController-updateDelay-uint256-[`updateDelay`] function. Bear in mind that access to this function is only accessible by the timelock itself, meaning this maintenance operation has to go through the timelock itself.
+
+[[access-management]]
+== Access Management
+
+Although xref:api:access.adoc#AccessControl[`AccessControl`] offers a more dynamic solution for permissioning your contracts, decentralized protocols tend become more complex after integrating new contract instances, requiring to keep track permissions in each one. This increases complexity of uniformly monitoring and updating permissions across the system.
+
+image::access-control-multiple.svg[Access Control multiple]
+
+These protocols often require a single main interface to manage all roles, while keeping the benefits of delaying access control operations and the same built-in rules for assigning and revoking roles.
+
+This is accomplished with an xref:api:access.adoc#AccessManager[`AccessManager`] instance; a contract to store multiple smart contract permissions. Instead of managing each contract separately, it allows managing access from a single contract, making your protocol easier to audit and control.
+
+image::access-manager.svg[Access Manager]
+
+The manager allows granting permissions to a role, scoped by contract (called target) and https://docs.soliditylang.org/en/v0.8.20/abi-spec.html#function-selector[function selector] (called function). In this model, each role is allowed to call multiple functions in different contracts. Similarly, a role can be granted to multiple addresses, and a single address can be granted with multiple roles.
+
+image::access-manager-functions.svg[Access Manager functions]
+
+=== Using `AccessManager`
+
+Contracts includes xref:api:access.adoc#AccessManager[`AccessManager`], it can be deployed and used out of the box. It sets an initial admin in the constructor, who will be allowed to perform management operations.
+
+For a fresh new contract, you must implement the xref:api:access.adoc#AccessManaged[`AccessManaged`] contract provided along with the manager for matching permissions. This contract should define an Access Manager instance as its xref:api:access.adoc#AccessManaged-authority--[authority]. 
```

Comment:
```suggestion
In order to restrict access to some functions of your contract, you should inherit from the xref:api:access.adoc#AccessManaged[`AccessManaged`] contract provided along with the manager. You will have to set the addess of the Access Manager instance (xref:api:access.adoc#AccessManaged-authority--[authority]) in the constructor. This will provide `restricted` modifier that you can use to protect any public-facing function. 
```